src/Stryker.ts

```diff
@@ -103,7 +101,7 @@ export default class Stryker {
     }
   }
 
-  private async initialTestRun(inputFiles: InputFile[]): Promise<any> {
+  private async initialTestRun(inputFiles: InputFile[]) {
```

Comment:
Is it really a good idea to remove this return type?
Future releases of typescript might require you to set this return type.